ngraph/python/CMakeLists.txt

```diff
@@ -16,30 +16,71 @@
 
 cmake_minimum_required (VERSION 3.13)
 
-project (pyngraph)
-
-include(ExternalProject)
-
-ExternalProject_Add(
-        pybind11
-        GIT_REPOSITORY "https://github.com/pybind/pybind11.git"
-        GIT_TAG "v2.5.0"
-        SOURCE_DIR "${CMAKE_CURRENT_BINARY_DIR}/pybind11"
-        CONFIGURE_COMMAND ""
-        BUILD_COMMAND ""
-        INSTALL_COMMAND ""
-)
+project (_pyngraph)
 
-set(BUILD_PY_IN "${CMAKE_CURRENT_SOURCE_DIR}/build_wheel.py.in")
-set(BUILD_PY    "${CMAKE_CURRENT_BINARY_DIR}/build_wheel.py")
-configure_file(${BUILD_PY_IN} ${BUILD_PY} @ONLY)
+include(FetchContent)
 
-add_custom_command(
-        DEPENDS pybind11
-        OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/dist/
-        POST_BUILD
-        WORKING_DIR ${CMAKE_CURRENT_BINARY_DIR}
-        COMMAND python ${BUILD_PY}
+FetchContent_Declare(
+  pybind11
+  GIT_REPOSITORY "https://github.com/pybind/pybind11.git"
+  GIT_TAG        "v2.5.0"
 )
 
-add_custom_target(python_wheel DEPENDS ngraph ${CMAKE_CURRENT_BINARY_DIR}/dist/)
+FetchContent_GetProperties(pybind11)
+if(NOT pybind11_POPULATED)
+  FetchContent_Populate(pybind11)
+  add_subdirectory(${pybind11_SOURCE_DIR} ${pybind11_BINARY_DIR})
+endif()
+
+find_package(ngraph REQUIRED)
+if(ngraph_FOUND)
+    message("ngraph version = {${ngraph_VERSION}}")
+endif()
+
+find_package(Python COMPONENTS Interpreter Development REQUIRED)
+
+if(PYTHON_FOUND)
+    set(PYTHON_VERSION python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR})
+    message("Python version={${Python_VERSION}}")
+else()
+    message(FATAL_ERROR "Python was not found!")
+endif()
+
+if(UNIX)
+    add_compile_options(
+            -fPIC
+            -Wformat
+            -Wformat-security)
+            
+    if(CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 4.8.5)
+        add_compile_options(-fstack-protector-strong)
+    else()
+        add_compile_options(-fstack-protector)
+    endif()
+
+    add_link_options(-z noexecstack)
+
+endif()
+
+if (APPLE)
+     add_link_options(-stdlib=libc++)
+endif()
+
+if (MSVC)
+     add_compile_options(/wd4996)
+endif()
+
+file(GLOB_RECURSE SOURCES src/pyngraph/*.cpp)
+pybind11_add_module(${PROJECT_NAME} MODULE ${SOURCES})
+target_include_directories(${PROJECT_NAME} PRIVATE src)
+target_link_libraries(${PROJECT_NAME} PRIVATE ngraph::ngraph)
+
+ie_cpack_add_component(pyngraph REQUIRED)
```

Comment:
it's not available if InferenceEngineDeveloperPackage is not used.

Maybe we need to use InferenceEngineDeveloperPackage ? In this case all SDL and other flags are provided and we don't have to use all options like `add_compile_options(/wd4996)`

Reply:
- Added InferenceEngineDeveloperPackage package
- wd4996 is required even InferenceEngineDeveloperPackage is included